### DISCUSSION TITLE
YT Help

### DISCUSSION THREAD

**Discussion by Efferon** [danielmiessler/Fabric#412] [Q&A]
I could use some help
FirstIf there is another discussion group where this should be posted let me know. 

Second I've installed fabric following the quick install process, and the sample "pbpaste | fabric --pattern summarize" works well. When I use the YT sample "yt --transcript https://www.youtube.com/watch?v=wPEyyigh10g | fabric --stream --pattern extract_wisdom" I get an error zsh: no matches found: https://youtube.com/watch?v=wPEyyigh10g

Even if I just run "yt --transcript https://www.youtube.com/watch?v=wPEyyigh10g" I get the same error.

Is there some process to activate a pyenv first?

**Comment by kkeeling** [comment]
Try putting quotes around the url: yt "https://youtube.com/watch?v=uXs-zPc63kM"

**Comment by zoikville** [comment]
Hi - I have the same issue.

yt --transcript "https://youtube.com/watch?v=uXs-zPc63kM" | fabric --stream --pattern extract_wisdom

do you need the YouTube key to use this? 
Error: YOUTUBE_API_KEY

**Reply by black-backdoor** [reply]
You need an YouTube API key for it to work, you can create one for free. All you need is a Google account. Here's a helpful tutorial to get you started: [Tutorial Link](https://developers.google.com/youtube/v3/getting-started)
